pkg/syncer/syncer.go

```diff
@@ -73,15 +73,58 @@ func (sc *SyncerConfig) ID() string {
 func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, importPollInterval time.Duration) error {
 	klog.Infof("Starting syncer for logical-cluster: %s, workload-cluster: %s", cfg.KCPClusterName, cfg.WorkloadClusterName)
 
+	kcpClusterClient, err := kcpclient.NewClusterForConfig(rest.AddUserAgent(rest.CopyConfig(cfg.UpstreamConfig), "kcp#syncer/v0.0.0"))
```

Comment:
Either TODO to fill in the version, or pull it from either `k8s.io/component-base/version` or `k8s.io/client-go/pkg/version`?

Reply:
Fixed